packages/a11y-testing/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/a11y-testing",
+  "version": "0.1.0",
```

Comment:
Is this intended to be published? If not you can add `"private": true`

Reply:
I am not sure about long term plan. @jurokapsiar please could you clarify? :)

Reply:
not now, just internal. maybe in the future